suite-native/module-settings/src/slice.ts

```diff
@@ -14,6 +15,7 @@ type SliceState = {
 
 const initialState: AppSettingsState = {
     colorScheme: 'system',
+    currency: fiatCurrenciesMap.usd,
 };
 
 export const appSettingsSlice = createSlice({
```

Comment:
Can you please add tests for this whole slice?

Reply:
https://github.com/trezor/trezor-suite/pull/5895/commits/48aeb4910f78534e93288e77a7d475d122aeceee